src/app/content/components/Page/contentDOMTransformations.ts

```diff
@@ -209,3 +210,18 @@ function moveFootnotes(document: Document, rootEl: HTMLElement, intl: IntlShape)
     sup.appendChild(link);
   }
 }
+
+const setLinksAttributes = (rootEl: HTMLElement) => {
+  for (const a of Array.from(rootEl.querySelectorAll('a'))) {
```

Comment:
is NodeList not iterable? i'm not sure you really need the `Array.from` here

Reply:
It can't be used in for of loop because it lacks Symbol.iterator() - according to my linting plugin, but it has `.forEach` method built it. I'll use it